### PR TITLE
Remove vulnerable depnedency warning

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -157,7 +157,7 @@ DEPENDENCIES
   webpacker
 
 RUBY VERSION
-   ruby 2.5.1p57
+   ruby 2.3.1p112
```
Comment (yannski) [inline Gemfile.lock]: ?

File: Gemfile.lock [hunk 2]
```diff
@@ -132,7 +132,7 @@ GEM
     thread_safe (0.3.6)
     tzinfo (1.2.5)
       thread_safe (~> 0.1)
-    webpacker (3.2.2)
+    webpacker (3.5.5)
```
Comment (yannski) [inline Gemfile.lock]: ?

File: Gemfile.lock (hunk 1 above)
Comment (Soulou) [reply Gemfile.lock]: My bad, my local docker image was old, it's getting rebuilt

**Comment by johnsudaar** [comment]
It does not build anymore @Soulou ...

**Comment by Soulou** [comment]
Yes I've seen thanks :D

**Pull request merged**